firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackActivity.java

```diff
@@ -60,22 +64,35 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   private void setupView() {
+    setTheme(R.style.Theme_Feedback);
     setContentView(R.layout.activity_feedback);
 
+    TextView senderText = findViewById(R.id.senderText);
+    senderText.setText(getString(R.string.sender, "tester@e.mail"));
+
     TextView infoTextView = this.findViewById(R.id.infoText);
     infoTextView.setText(infoText);
     infoTextView.setMovementMethod(LinkMovementMethod.getInstance());
-    Button submitButton = this.findViewById(R.id.submitButton);
-    submitButton.setOnClickListener(this::submitFeedback);
+
+    findViewById(R.id.backButton).setOnClickListener(v -> finish());
+
+    Button sendButton = this.findViewById(R.id.sendButton);
+    sendButton.setOnClickListener(this::submitFeedback);
 
     Bitmap thumbnail = screenshotUri == null ? null : readThumbnail();
     if (thumbnail != null) {
-      ImageView screenshotImageView = this.findViewById(R.id.thumbnail);
+      ImageView screenshotImageView = this.findViewById(R.id.screenshotImageView);
       screenshotImageView.setImageBitmap(thumbnail);
+      CheckBox checkBox = findViewById(R.id.screenshotCheckBox);
+      checkBox.setChecked(true);
+      checkBox.setOnClickListener(
+          v -> screenshotImageView.setVisibility(checkBox.isChecked() ? VISIBLE : GONE));
     } else {
       LogWrapper.getInstance().e(TAG, "No screenshot available");
```

Comment:
Will this text get translated?

Reply:
No. But none of the log messages are. They are not shown to the end user.